src/08-accessing.js

```diff
@@ -148,7 +164,11 @@ function getCityAndState(person) {}
  *  getFullAddress(person);
  *  //> "697 Pine Drive 2A, Staten Island, NY, 10306"
  */
-function getFullAddress(person) {}
+function getFullAddress(person) {
+
+    return person.address.street +" "+ person.address.unit +", "+ person.address.city +", "+ person.address.state +", "+ person.address.zip;
```

Comment:
We can move this down to a multi line string as we are using backticks.  It makes the line easier to read.